docs/Interoperability_tests_guide/Using_API_Jenkins_integration.md

```diff
@@ -0,0 +1,310 @@
+               if result.status_code != 200:
+                       print('Could not read the tests status.')
+                       print('Server returned: ' + str(result.status_code))
+                       print('Error message: ' + str(result.json()['error']))
+                       exit(1)
+               tests_running = result.json()["waitingForExecution"]
+               time.sleep(15)
+
+       for test in result.json()['failed']:
+           a = ET.SubElement(root, "testcase", classname="interop", name=test)
+           ET.SubElement(a, "failure", type="failure")
+
+       for test in result.json()['passedWithWarning']:
+           b = ET.SubElement(root, "testcase", classname="interop", name=test)
+           ET.SubElement(b, "failure", type="warning")
+
+       for test in result.json()['passedSuccessfully']:
+           ET.SubElement(root, "testcase", classname="interop", name=test)
+
+       tree = ET.ElementTree(root)
+       tree.write("report.xml")
+       ```
+
+          - `DEVICE_NAME` - type the endpoint name of your device.
+          - `INSTALLATION_URL` - provide the URL of your Coiote DM installation.
+          - `INSTALLATION_API_PORT` - provide the port for communication with the API. The default value is `8087`.
+          - `CREDENTIALS` - provide user name and password of your Coiote DM user account.
+          - `TEST_NAMES` - type the names of the test cases that you want to execute on the device.
+    - Save the script as a `.py` file (using the filename specified in the Jenkinsfile in the previous step) and upload it to your project repository.
+
+2. Create a pipeline for your project:
+    - Go to your Jenkins account and in the **Dashboard** view, select **New Item** from the menu on the left.
+    - Enter a name for your pipeline, select **Multibranch Pipeline** and confirm by clicking **OK**.
+![Creating a multibranch pipeline](images/image112.png "Creating a multibranch pipeline")
+3. Configure your pipeline:
+    - Go to your newly created pipeline and select **Configure** from the menu on the left.
+    - In the **Branch Sources** section, select the **Git** option and provide the following:
+        - **Project Repository** - enter the URL address of your project repository that hosts the `Jenkinsfile` and the python script file from step 2.  
+![Configuring a multibranch pipeline](images/image115.png "Configuring a multibranch pipeline")
+        - **Credentials** - add the user name and password of your GitLab account.
+    - In the **Build Configuration** section, select the **by Jenkinsfile** mode from the drop-down list and provide the GitLab path to the `Jenkinsfile` from step 1 (if the file is located in the GitLab root folder, it is enough to type `Jenkinsfile`)
+![Build configuration - multibranch pipeline](images/image116.png "Build configuration - multibranch pipeline")
+    - Click **Save**.
+
+### GitLab - configure and run pipeline
```

Comment:
Could you extract Gitlab section to a separate point? Adding this to Jenkins multibranch makes this tutorial chaotic.

Reply:
done